compiler/src/main/java/bullet/impl/ComponentProcessingStep.java

```diff
@@ -224,4 +236,31 @@ private boolean isVisibleFrom(Element target, PackageElement from) {
         throw new AssertionError();
     }
   }
+
+  class TypeMirrorWrapper {
```

Comment:
Oh, BTW, it should be possible to replace this with a Guava `Equivalence.Wrapper` using `MoreTypes.equivalence()` as the underlying `Equivalence`; i.e. `MoreTypes.equivalence().wrap(someTypeMirror)`.

It would behave correctly wrt `hashCode` in case two distinct `TypeMirror` objects represent the same type; which this `TypeMirrorWrapper` doesn't guarantee.